### ISSUE TITLE
API v2 Support

### ISSUE THREAD

**Issue by moth** [incraigulous/prismic-toolkit#6]
Good stuff incraigulous. We're just getting started with Prismic and this seems a lot better than the bundled SDK.

Do you have any plans to update for the v2 API?

**Comment by incraigulous** [comment]
Yes, this is on my radar, but I'm not sure how soon it will happen. I will upgrade it to v2 of the API before the package hits it's 1.0 release for sure.
